daemon/src/lib.rs

```diff
@@ -420,13 +420,12 @@ where
             .context("Price feed not available")?
             .context("No quote available")?;
 
-        let current_price = Price::new(latest_quote.for_taker())?;
         let quote_timestamp = latest_quote
             .timestamp
             .format(&time::format_description::well_known::Rfc3339)
             .context("Failed to format timestamp")?;
 
-        tracing::debug!(%order_id, %current_price, %quote_timestamp, "Proposing settlement of contract");
+        tracing::debug!(%order_id, %quote_timestamp, "Proposing settlement of contract");
```

Comment:
would be good to keep the price information in there.

Reply:
We cannot easily do the decision of bid/ask in here because we don't have access to the database.
I can move the comment to `taker_cfd` and record the proposal price, *or* just record bid and ask here. Ideas?

Reply:
I would just print both?

Reply:
we can, but then we might not know which one was actually used unless we lookup the position of the `Cfd` in the database. If that is OK then I'm happy to add both :)

Reply:
I opted for moving the tracing statement and print all three prices, that should give us the best information.
This adds a bit of indirection, because we pass on the quote timestamp, but I think that is acceptable given that we get a better log statement. 
The log statement is also printed at a more precise point in time, because there were failure scenarios in between the the log statement and the actual settlement start. Now we print right before we actually start the settlement.